catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/security/AccessControlPolicyExtensionTest.java

```diff
@@ -130,6 +136,66 @@ public void testAccessAdminOfAclAlwaysImplied() {
     assertThat(after, is(Collections.emptyList()));
   }
 
+  @Test
+  public void testLackingIndividualReadImpliesNone() {
+    List<Permission> before = ImmutableList.of(INDIVIDUALS_READ_ONLY);
+
+    CollectionPermission subject =
+        subjectFrom(
+            makePermission(
+                Constants.EMAIL_ADDRESS_CLAIM_URI,
+                ImmutableSet.of("non-existent-email@connexta.com")));
```

Comment:
✏️ Make `non-existent-email@connexta.com` a constant

Reply:
It's only used once, so I didn't see value in this.